weather_app.py

```diff
@@ -21,10 +24,26 @@ def display_weather(weather_data):
         print(f"Weather in {weather_data['name']}: {description}")
         print(f"Temperature: {temperature}°C, Humidity: {humidity}%")
 
+def display_forecast(forecast_data):
```

Comment:
I thought I had reviewed this code already? wondering why it's part of this diff?